types/index.d.ts

```diff
@@ -5205,6 +5206,13 @@ export interface CookiesNsCommands<ReturnType = unknown> {
   ): Awaitable<IfUnknown<ReturnType, this>, null>;
 }
 
+export interface FirefoxNsCommands<ReturnType = unknown> {
+  getContext(): Awaitable<IfUnknown<ReturnType, this>, 'content' | 'chrome'>;
+  setContext(ctx: 'content' | 'chrome' | PromiseLike<'content' | 'chrome'>): Awaitable<IfUnknown<ReturnType, this>, null>;
```

Comment:
Since `'content' | 'chrome'` is being used at multiple places, it can be defined as a separate type and used here:

```ts
type FirefoxContext = 'content' | 'chrome';
```

Reply:
Any specific reason to use `PromiseLike`?